auth.go

```diff
@@ -139,17 +141,46 @@ func WithDIDResolver(resolver packers.DIDResolverHandlerFunc) VerifierOption {
 	}
 }
 
+// WithEthereumResolvers set the ethereum resolvers to use. It will overwrite the
+// default ones.
+func WithEthereumResolvers(resolvers map[int]eth.Resolver) VerifierOption {
+	return func(opts *verifierOpts) {
+		opts.ethResolvers = resolvers
+	}
+}
+
 type verifierOpts struct {
-	docLoader   ld.DocumentLoader
-	ipfsCli     schemaloaders.IPFSClient
-	ipfsGW      string
-	didResolver packers.DIDResolverHandlerFunc
+	docLoader    ld.DocumentLoader
+	ipfsCli      schemaloaders.IPFSClient
+	ipfsGW       string
+	didResolver  packers.DIDResolverHandlerFunc
+	ethResolvers map[int]eth.Resolver
```

Comment:
ethResolvers is needed in DefaultZKPUnpacker constructor.  https://github.com/iden3/iden3comm/blob/14c597734e9db3b77133adf979af5c84abcbfb15/packers/zkp.go#L166 

In some way, I need to inject it in the JWZ unpacker, via the Unpack function or via the constructor. I decided to go via unpack parameter to do not complicate JWZ packer constructor.